nextflow.config

```
@@ -45,6 +45,10 @@ params {
   results_bucket = "s3://openscpca-nf-workflow-results-staging"
   sim_bucket = "s3://openscpca-test-data-release-staging"
   project = "all"
+
+  // URIs to reference files
+  gtf_file = 's3://scpca-references/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.gtf.gz'
```

To get ahead of an upcoming change.
```suggestion
  gtf_file = 's3://scpca-nf-references/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.gtf.gz'
```